src/components/ScaleLattice.vue

```diff
@@ -0,0 +1,351 @@
+<script setup lang="ts">
+import type { Interval, Scale } from "scale-workshop-core";
+import { computed, onMounted, type ComputedRef, ref, onUnmounted } from "vue";
+import { monzoEuclideanDistance } from "@/utils";
+import { kCombinations } from "xen-dev-utils";
+const props = defineProps<{
+  scale: Scale;
+  heldScaleDegrees: Set<number>;
+}>();
+
+type Coord = { x: number; y: number };
+type Node = {
+  x: number;
+  y: number;
+  label: string;
+  index: number;
+  fill: string;
+};
+
+type Coords = Coord[];
+
+const coords: Coords =
+  // Coords for every prime up to 23, indexes correspond to monzo indexes
+  // based on Kraig Grady's coordinate system https://anaphoria.com/wilsontreasure.html
+  [
+    {
+      x: 0,
+      y: 0,
+    },
+    {
+      x: 40,
+      y: 0,
+    },
+    {
+      x: 0,
+      y: -40,
+    },
+    {
+      x: 13,
+      y: -11,
+    },
+    {
+      x: -14,
+      y: -18,
+    },
+    {
+      x: -8,
+      y: -4,
+    },
+    {
+      x: -5,
+      y: -32,
+    },
+    {
+      x: 7,
+      y: -25,
+    },
+    {
+      x: 20,
+      y: -6,
+    },
+  ];
+
+const container = ref<HTMLDivElement | null>(null);
+
+// TODO improve
```

Comment:
Prefer not to leave comments in production code. If left in, prefer to reference the issue that explains what needs to be done like #38.